migrate/src/main/scala/migrate/ScalaMigrat.scala

```diff
@@ -96,19 +107,16 @@ object Main {
     } yield ()
 
   private def buildMigrationFiles(
-    sources: Seq[AbsolutePath],
-    classpath: Classpath,
-    targetRoot: AbsolutePath,
-    toolClasspath: Classpath,
-    compilerOptions: Seq[String]
+    unmanagedSources: Seq[AbsolutePath],
+    scalafixSrv: ScalafixService
```

Comment:
I guess this `scalafixSrv` is useless because it is accessible from the class.

Reply:
yes thanks !